rules.org

```diff
@@ -1,14 +1,69 @@
 #+SETUPFILE: org-theme/setup/short.setup
-
-#+AUTHOR: Manoel Vilela
-#+DATE: <2018-02-10 Sat>
+#+AUTHOR: Manoel Vilela, Lucas S. Vieira
+#+DATE: <2020-09-20 dom>
 #+TITLE: Regras
 
+* Que grupo é esse?
+
+A *Common Lisp Brasil* é a principal comunidade brasileira a respeito da
+linguagem de  programação Common Lisp, onde  discute-se principalmente
+sobre  programação  nessa  linguagem,   mas  também  admite-se  outros
+assuntos relacionados, desde que /com parcimônia/.
+
+* Quais são as regras?
+
+As  "regras"  a  seguir  são  linhas  gerais  para  garantir  uma  boa
+convivência na nossa comunidade de Common Lisp.
+
+Não temos  a prática de  reforçar regras restritivas. Ao  invés disso,
+esperamos que  os participantes  possam usufruir de  um bom  espaço de
+discussão onde majoritariamente discuta-se Common Lisp.
+
+Alguns  pontos de  convivência estão  listados a  seguir. Na  falta ou
+dubiedade de  algum tópico, esperamos  que o membro possa  confiar nas
+decisões  dos moderadores,  que  lidarão com  a  situação.  Caso  seja
+necessário, esta página poderá também ser modificada no futuro.
+
+1. Aja  com *boa-fé*. Escreva  com respeito e seja  respeitado. Ademais,
+   nunca aja como  se algum comentário tivesse a  intenção de ofender;
+   se esse for o caso, a moderação agirá para resolver a situação.
+2. Não  poste /spam/, e não  inicie assuntos que não  tenham relação com
+   Common Lisp, outros dialetos de Lisp ou Tecnologia em geral.
+3.  Somos muito receptivos a  discussões sobre outros dialetos de Lisp
+   como Scheme,  Clojure, LFE, etc.   Mas cuide  para que o  grupo não
+   "deixe de ser" um grupo de Common Lisp no processo.
+4.  Os moderadores  são  seus amigos  e tentam  agir  da melhor  forma
```

Comment:
Essa regra está correlacionada com a (1), acredito que é desnecessária. Seria bom condensar.

Reply:
O @renanroberto perguntou via chat se seria legal deixar claro que permitimos discussões sobre outros dialetos. Gosto da ideia, mas não quero que vire um canal de propaganda de outros Lisps. Talvez dê pra colocar isso de forma sucinta na (1).

Reply:
Só pra discorrer um pouco mais sobre o tópico: Penso que seria interessante deixar isso claro porque a priori não parece óbvio que discussão sobre outros lisps é bem vinda. Alguém pode ter a impressão de que existe uma relação de inimizade entre os dialetos. Entretanto realmente é bom deixar claro o foco do grupo

Reply:
Nesse comentário me refiro a regra (4) que começa na linha 35.

Reply:
acredito que se forçarmos a discussão a não sair de Common Lisp, pode ficar talvez um pouco sufocante/bitolante. Em algumas situações é interessante comparar, ver o que há ao lado. Claro, o foco é Common Lisp.
Me vem à cabeça o final do "Land of Lisp", onde ele apresenta brevemente na estorinha os outros Lisps.

Reply:
Sobre a regra sobre afinidade com outros dialetos, beleza, vocês argumentaram ter um bom motivo pra estar nas regras. Então vamos deixá-la com seu espaço.

Reply:
Vou condensar a regra 4 com a regra 1 então, deletando essa primeira. Vi que o @parjanya fez mais sugestões, então vou prosseguir com a revisão.